extensions/infinispan-client/deployment/src/main/java/io/quarkus/infinispan/client/deployment/InfinispanClientProcessor.java

```diff
@@ -62,6 +63,7 @@
     private static final String META_INF = "META-INF";
     private static final String HOTROD_CLIENT_PROPERTIES = META_INF + File.separator + "hotrod-client.properties";
     private static final String PROTO_EXTENSION = ".proto";
+    private static final String SASL_SECURITY_PROVIDER = "com.sun.security.sasl.Provider";
```

Comment:
We use this in https://github.com/quarkusio/quarkus/blob/57d1dae99a635b32951d7e1594f0969fc61dceb1/extensions/kafka-client/deployment/src/main/java/io/quarkus/kafka/client/deployment/KafkaProcessor.java#L118 as well. Would it make sense to gather such definitions in a single place?

Reply:
I personally don't think it's worth it.